### PR TITLE
#1149 Invert the LDAP example on the README page

### PR DESCRIPTION
**Pull request by wpater**
Content in README.md rearranged and improved.

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.
Not needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +37,39 @@ jenkins:
...
+
+  slaveAgentPort: 50000
+  agentProtocols:
+    - "jnlp2"
+tool:
+  git:
+    installations:
+      - name: git
+        home: /usr/local/bin/git
+unclassified:
```
Comment (oleg-nenashev) [inline README.md]: Maybe we want to drop this "unclassified" section to get better look and feel. IIUC the smtpHost issue is fixed anyway, CC @alecharp

Comment (wpater) [reply README.md]: I can do that. It's my first commit here so I just need confirmation from someone to remove whole `unclassifield` node :).

Comment (oleg-nenashev) [reply README.md]: +1 from me, it is just an example

Comment (casz) [reply README.md]: 👍 we have a demo, so we can cleanup the example in the main README :)

Comment (oleg-nenashev) [reply README.md]: I think that a short sample is still useful. The size is yet to be determined, but we go in a right direction

Comment (wpater) [reply README.md]: Removed.

**Comment by oleg-nenashev** [comment]
LGTM Bot hangs due to whatever reason

**Pull request merged**